models/comments.model.js

```diff
@@ -26,4 +26,21 @@ function insertCommentByArticleId(articleId, body) {
     });
 }
 
-module.exports = { selectCommentsByArticleId, insertCommentByArticleId };
+function deleteCommentById(commentId) {
+  return db
+    .query(`DELETE FROM comments WHERE comment_id = $1 RETURNING*;`, [
```

Comment:
Nitpick, but usually you'd have a space in between the RETURNING and the *  (pgsql reads it in the same way though)